### PR TITLE
[SOW MS3]: Enable test_batchnorm_cudnn_nhwc 

### PR DESCRIPTION
**Pull request by bmedishe**
test_nn.py::TestNN::test_batchnorm_cudnn_nhwc test passes by cherry picking change from PR [skip miopen batchnorm if channels_last (NHWC)](https://github.com/ROCmSoftwarePlatform/pytorch/pull/1025)

### CODE REVIEW

**Comment by jithunnair-amd** [comment]
jenkins retest this please

**Comment by jithunnair-amd** [comment]
@bmedishe CI is down. Would you be able to post a snippet from your local run log that show this test passing? Please mention docker image used as well.

**Comment by bmedishe** [comment]
> @bmedishe CI is down. Would you be able to post a snippet from your local run log that show this test passing? Please mention docker image used as well.

image

**Comment by bmedishe** [comment]
I can say that the build and this one test are fine.
